src/main/resources/assets/api-docs.yaml

```diff
@@ -1868,7 +1868,7 @@ paths:
           description: The election or the DAR couldn't be found.
     post:
       summary: createDataRequestElection
-      description: Creates an Election for the DAR identified by the ID.
+      description: Creates an access and RP Election for the DAR identified by the ID.
```

Comment:
The original language is preferable in my mind. On a technical level, we are creating a DAR election. In some cases, we also create an RP election, but not always. In some cases, we also create a Data Owner election, but not always. Describing that internal complexity here would require a good deal more description than we have.